.yarn/patches/yoga-layout-prebuilt-npm-1.10.0-855b15449f.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/yoga-layout/build/Release/nbind.js b/yoga-layout/build/Release/nbind.js
```

Comment:
Any context on this file?

Reply:
It's a small one line patch for yoga dependency. It will be automatically applied by yarn on `yarn install`.